src/initializers/honeybadger.ts

```diff
@@ -1,9 +1,19 @@
-export default async (config) => {
+export default async config => {
   if (config.honeybadger.apiKey) {
-    const honeybadger = await import('honeybadger');
+    const honeybadger = await import('@honeybadger-io/js');
     honeybadger.configure({
+      apiKey: config.honeybadger.apiKey,
       environment: config.app.env,
       developmentEnvironments: config.honeybadger.developmentEnvironments
     });
+    honeybadger.beforeNotify(function (notice) {
```

Comment:
🔧  We set the fingerprint on honeybadger-appender.ts. Please move the logic there.

Reply:
I added it there because it seems like the proposed place from [hb documentation](https://docs.honeybadger.io/lib/javascript/guides/customizing-error-grouping/).
Backtrace also exists on callback, not in appender. We could move all fingerprint calculation, but I preferred to have it calculated on appender and "enhance" it on callback

Reply:
I would put it in one place (appender) and use the error and error.stacktrace to enhance it. (I guess this is what backtrace does internally but this way you won't have to read the lib's code to understand what is being logged and if it is nullable etc.)

Reply:
I see that honeybadger-js is using https://www.npmjs.com/package/stack-trace so the way you do it is the easier one without also importing this lib. Just remove the console.logs then and leave it as is.

Reply:
console.logs are removed

Reply:
Checked the maestro a bit. I am worried we will keep having new error groups for known errors, e.g. check https://app.honeybadger.io/projects/50040/faults?q=component%3Arabbit-queue+action%3Aparse+environment%3Aofficedroid&sort=last_seen_desc.
With every deploy that changes the line of an error we will get a different group.